packages/lodestar/package.json

```diff
@@ -91,6 +91,7 @@
     "interface-datastore": "^5.1.2",
     "it-all": "^1.0.2",
     "it-pipe": "^1.1.0",
+    "jwt-simple": "^0.5.6",
```

Comment:
Consider pinning a specific version of `jwt-simple` to prevent dependency attacks

Reply:
done. :+1: https://github.com/ChainSafe/lodestar/pull/3777/commits/8f8197d55d2021f04fa1927cd6bd76c9b4db1c5d